packages/chip-input/src/ux-chip.css

```diff
@@ -0,0 +1,48 @@
+ux-chip {
+  display: inline-flex;
+  align-items: center;
+
+  font-size: 14px;
+
+  height: 32px;
+  border-radius: 100px;
+
+  background-color: var(--ux-theme--chip-background);
+  color: var(--ux-theme--chip-foreground);
+}
+
+  ux-chip:focus {
+    outline: none;
+    box-shadow: var(--design-elevation4dp);
+  }
+
+  ux-chip > span {
+    margin: 0 12px;
+  }
+
+    ux-chip > span.close {
+      display:none;
+    }
+
+  ux-chip[deletable] > span {
+    margin-right: 0;
+  }
+
+    ux-chip[deletable] > span.close {
+      display: inline-flex;
+      justify-content: center;
+      align-items: center;
+      margin: 0 4px;
+      color: var(--ux-theme--chip-delete-foreground);
+      background-color: var(--ux-theme--chip-delete-background);
```

Comment:
I'm wondering about the variable naming convention. Why two dashed between ux-theme and chip but above only one dash between design and elevation4dp?

Reply:
This was update to `--ux-design--` as the prefix to match up with the `--ux-theme--` prefix.